packages/support/lib/image-util.js

```diff
@@ -1,14 +1,32 @@
-import sharp from 'sharp';
+let _sharp;
+
+/**
+ * @returns {import('sharp')}
+ */
+function requireSharp() {
+  if (!_sharp) {
+    try {
+      _sharp = require('sharp');
+    } catch (err) {
+      throw new Error(
+        `Cannot load the 'sharp' module needed for images processing. ` +
```

Comment:
maybe add a line like `Automatic install of sharp failed during Appium npm install`  here?